webpack.common.js

```diff
@@ -53,7 +53,7 @@ module.exports = {
 	plugins: [
 		new VueLoaderPlugin(),
 		new StyleLintPlugin(),
-		new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+		new webpack.IgnorePlugin(/^\.\/locale$/, /^\.\/locales$/, /moment$/),
```

Comment:
I'm guessing this is not a proper regex.
Just this should be enough to reduce the total size :)

Also, please use a babel.config.js file instead of having the babel confs in the webpack file :)

Reply:
I tried changing it to:
```
		new webpack.IgnorePlugin(/^\.\/locale(s)?$/, /(@fullcalendar)$|(moment)$/),
```

but that only brings it down to 1.98, not more :/

Reply:
The webpack output should display the other chunks if it is working :)
https://travis-ci.com/nextcloud/contacts/jobs/248200403#L2009